### PR TITLE
YLP-1800 / YLP-1799 Implement Power Saving objects upload (+documentation update) 

### PR DESCRIPTION
**Pull request by CedricPerotto**

### CODE REVIEW

File: docs/UpdatedData.md [hunk 1]
```diff
@@ -268,12 +272,12 @@ The below fields are mandatory if `alarmType` is set to `handset`. They remain o
...
 ## Zen mode , Confidential mode && Sensor Warmup
 
-Leveraging the `deviceEvent` type and creating 2 new subTypes with the same structure: `zen` and `confidential`.
+Leveraging the `deviceEvent` type and creating new subTypes with the same structure: `zen`, `confidential`, `warmup` and `loopMode` .
 
-- `subType`: `zen | confidential | warmup`
-- `duration`: is a structured object that gives the duration of the event. __This field is mandatory__.
+- `subType`: `zen | confidential | warmup | loopMode`
+- `duration`: is a structured object that gives the duration of the event. __This field is mandatory for the subType `zen`, `confidential`, and `warmup`.__ It is optional for loopMode subType (the duration is updated at the end of the event).
 - `eventId`: unique ID provided by the client that is used to link stop and start events. __This ID is mandatory__.
```
Comment (franck-fourel) [inline docs/UpdatedData.md]: should we remove the eventId as we have activated the guid for all datum?

**Review by franck-fourel — COMMENTED** [review]
question about the eventId that could be deprecated, so that we switch to guid

File: docs/UpdatedData.md (hunk 1 above)
Comment (CedricPerotto) [reply docs/UpdatedData.md]: Old dblXX will keep pushing eventId field, and some front end operations (0 duration physical activities interpreted as deleted ones for example) also use the eventId to filter some data.

Comment (CedricPerotto) [reply docs/UpdatedData.md]: switch to guid done and implemented here (when **guid** is present and not empty **guid** overrides **eventId**): 
https://github.com/mdblp/platform/pull/104

File: docs/UpdatedData.md [hunk 2]
```diff
@@ -241,7 +247,7 @@ In the below example, The 2 objects are sent in 2 separate requests. The first o
 Leveraging the `deviceEvent` type with the already defined `alarm` subType. We add couple of fields to get more details on alarms and acknowledgement. 
 
 The below fields are mandatory if `alarmType` is set to `handset`. They remain optional for any other values.
-- `eventId`: unique Id of the event generated by the client system. This ID will be used to reconciliate data for the same event. _Maximum length is 64 characters_.
+- `guid` (replacing `eventId` deprecated field): unique Id of the event generated by the client system. This ID will be used to reconciliate data for the same event. _Maximum length is 64 characters_.
```
Comment (Nas84) [inline docs/UpdatedData.md]: this ID will be used with combination of DeviceId , userId to reconciliate data for the same event

**Review by Nas84 — APPROVED** [review]
LGTM, just a minor feedback on the doc regarding guid, just to indicate it is unique inside a terminal

File: docs/UpdatedData.md (hunk 2 above)
Comment (CedricPerotto) [reply docs/UpdatedData.md]: Done

**Pull request merged**